networking_monitoring.prolific

```diff
@@ -97,6 +97,99 @@ L: security
 
 ---
 
+Inspect your c2c packets with tcpdump
+
+### What?
+Sometimes a user says "container to container networking is broken! AppA can't talk to AppB". After making sure that they have c2c policies set, the next thing you might do is use tcpdump to inspect the packets.
```

Comment:
Might be worth defining `c2c`. Some students (e.g. designers or new Pivots) might not have the context.